rockcraft.yaml

```diff
@@ -0,0 +1,33 @@
+name: sdcore-gui
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+version: "0.1"
+summary: SD-Core GUI
+description: |
+  A graphical user interface for SD-Core.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  sdcore-gui:
+    override: replace
+    command: node client/.next/standalone/server.js
+    startup: enabled
+
+parts:
+  frontend:
+    plugin: npm
+    source: https://github.com/canonical/sdcore-gui.git
+    source-tag: 5aab82fe519ba73539bd492596d0de7b1f42f3c3
```

Comment:
This is a source-commit, not really a source tag

Reply:
Addressed.